src/markdown-pages/docs/managers/docker-manager2.md

```diff
@@ -0,0 +1,337 @@
+
+## <a name="limitations"></a>Limitations
+
+The Docker Manager supports only AMD64 platforms. It is planned to expand the capability to S390x. <br><br> The Docker Manager currently supports only a single Docker Engine.  It is planned to allow multiple Docker Engines to be configured. 
+
+# <a name="configuring"></a>Configuring the Docker Manager
+
+[Would it make any sense to give someone an idea / link to getting started with the Manager eg any steps / thought processes that would be a good way of understanding how to incorporate the Manager into your test. I guess this might be too difficult given the number of scenarios but perhaps there are some steps that might be common across all scenarios?]
+
+## <a name="methods"></a>Methods
+
+The following methods are available with the Docker Manager:
+
+<details>
+<summary>Method1</summary>
+
+Method1 does xyz. Use Method1 with the following parameters if you want to do this task. 
+[If there is another method with the same name but different parameters, then say in what instance you would call one or the other.]
+
+| Parameter: | Description |
+| --------------------------------------- | :------------------------------------- |
+| `parameter1` |  Description of parameter1 and how to use it |
+| `parameter2` |  Description of parameter2 and how to use it |
+| `parameter3` |  Description of parameter3 and how to use it |
+
+For more information, see the <a href="https://javadoc.galasa.dev/dev/galasa/docker/package-summary.html">Javadoc documentation</a>. <br>
+
+</details>
+
+<details>
+<summary>Method2</summary>
+
+Method1 does xyz. Use Method2 with the following parameters if you want to do this task. 
+
+| Parameter: | Description |
+| --------------------------------------- | :------------------------------------- |
+| `parameter1` |  Description of parameter1 and how to use it |
+| `parameter2` |  Description of parameter2 and how to use it |
+| `parameter3` |  Description of parameter3 and how to use it |
+
+For more information, see the <a href="https://javadoc.galasa.dev/dev/galasa/docker/package-summary.html">Javadoc documentation</a>. <br>
+
+</details>
+
+<details>
+<summary>Method3</summary>
+
+Method1 does xyz. Use Method3 with the following parameters if you want to do this task. 
+
+| Parameter: | Description |
+| --------------------------------------- | :------------------------------------- |
+| `parameter1` |  Description of parameter1 and how to use it |
+| `parameter2` |  Description of parameter2 and how to use it |
+| `parameter3` |  Description of parameter3 and how to use it |
+
+For more information, see the <a href="https://javadoc.galasa.dev/dev/galasa/docker/package-summary.html">Javadoc documentation</a>. <br>
+
+</details>
+
+
+## <a name="annotations"></a>Annotations
```

Comment:
can we call this Annotation provided by this manager?

Reply:
Have updated the subheading